pkgs/development/python-modules/cddlparser/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  # dependencies
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "cddlparser";
+  version = "0.5.0";
+  format = "pyproject";
```

`format = "..."` has been replaced with the following:

```suggestion
  pyproject = true;
```